src/TransitionListAwareTrait.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace Dhii\State;
+
+use Dhii\Util\String\StringableInterface as Stringable;
+use Exception as RootException;
+use InvalidArgumentException;
+use Traversable;
+
+/**
+ * Common functionality for objects that are aware of a list of transitions.
+ *
+ * @since [*next-version*]
+ */
+trait TransitionListAwareTrait
```

Comment:
Inconsistent naming with `_getTransitions()`.

Reply:
True. I was reluctant to use the name `TransitionsAwareTrait` due to the fact that a singular version exists, i.e. `TransitionAwareTrait`.